metabrainz/templates/index/contact.html

```diff
@@ -87,22 +87,31 @@ <h3>{{ _('Forums') }}</h3>
     answer any questions you may have.', forums_url='https://community.metabrainz.org/') }}
   </p>
 
-  <h3>{{ _('IRC') }}</h3>
+  <h3>{{ _('Chat') }}</h3>
   <p>
-    {{ _('The MetaBrainz/MusicBrainz team, as well as MusicBrainz supporters, hang out in the
-    <strong><a href="%(irc_url_mb)s">#musicbrainz</a></strong> and
-    <strong><a href="%(irc_url_meb)s">#metabrainz</a></strong>
-    channels on IRC chat at irc.libera.chat. If you don\'t have an IRC client
-    installed, consider using this
-    <a href="%(irc_webchat_url)s">webchat client</a>
-    instead. Please feel free to browse the
-    <a href="%(chatlogs_url)s">IRC log archives</a>.',
+    {{ _('The MetaBrainz/MusicBrainz/ListenBrainz team and community hang out
+    on our Matrix, IRC and Discord channels. Visit the
+    <a href="https://musicbrainz.org/doc/Communication/ChatBrainz">ChatBrainz</a>
+    page and join the chat. You can also browse the
+    <a href="%(chatlogs_url)s">chat log archives</a>.',
     irc_url_mb='ircs://irc.libera.chat:6697/musicbrainz',
     irc_url_meb='ircs://irc.libera.chat:6697/metabrainz',
     irc_webchat_url='https://kiwiirc.com/nextclient/irc.libera.chat/#musicbrainz,#metabrainz',
     chatlogs_url='http://chatlogs.metabrainz.org/') }}
   </p>
 
+  <h3>{{ _('Social Media') }}</h3>
+  <p>
+    {{ _('Nobody\'s perfect. You can follow us on these social channels:') }}
+     <ul>
+      <li>{{ _('<a href="https://blog.metabrainz.org/">Blog</a>') }}</li>
+      <li>{{ _('<a href="https://x.com/MetaBrainz">Twitter/X</a>') }}</li>
+      <li>{{ _('<a href="https://mastodon.social/@metabrainz">Mastodon</a>') }}</li>
+      <li>{{ _('<a href="https://bsky.app/profile/metabrainz.org">Bluesky</a>') }}</li>
+      <li>{{ _('<a href="https://www.instagram.com/MetaBrainz/">Instagram</a>') }}</li>
```

Comment:
Ideally these should also be done with variables such as `blog_url`, `mastodon_url`, etc - that way if the URL changes later (say, because some idiot buys Bluesky and renames it Y) it won't change the actual string translators will see.

Alternatively (and possibly even better?) you could just take the URL bit out of the translatable bit and leave just the title, since that seems to be what we already do in the footer?

Reply:
You had left the quotes in the wrong place, but I fixed that now :)